ols/app/endpoints/ols.py

```diff
@@ -109,7 +110,7 @@ def conversation_request(
 
     store_conversation_history(user_id, conversation_id, llm_request, response)
 
-    if config.ols_config.user_data_collection.transcripts_disabled:
+    if config_manager.get_ols_config().user_data_collection.transcripts_disabled:
```

Comment:
if we use Pydantic approach (w/o getters), most of the code won't need to change and this PR might be more minimalistic